dist/add-markers.js

```diff
@@ -318,7 +325,7 @@ Fliplet.InteractiveMap.component('add-markers', {
           id = _ref.id;
       return "".concat(name, " \u2014 [").concat(id, "]");
     },
-    chooseExistingData: function chooseExistingData() {
+    configureDs: function configureDs() {
```

Comment:
```suggestion
    configureDataSources: function configureDataSources() {
```

Reply:
Shorthands are bad, especially when the shorthand gets used in a distributed manner.

```js
function (str) {
  // res as a shorthand for result is tolerable here because it's used within a small area of code
  var res = str.replace('a', 'b');
  return res;
}
```

```js
function ds() {
  // ...
}

// 5000 lines later

ds(); // What's this?
```